client/src/App.tsx

```diff
@@ -24,24 +25,24 @@ function App(): JSX.Element {
         <SnackBarProvider>
           <AuthProvider>
             <SocketProvider>
-              <NavBar />
-
-              <Switch>
-                <Route exact path="/login" component={Login} />
-                <Route exact path="/signup" component={Signup} />
-                <Route exact path="/landing-page" component={LandingPage} />
-                <Route path="/profile" component={Profile} />
-                <Route exact path="/my-jobs" component={MyJobs} />
-                <Route exact path="/my-sitters" component={MySitters} />
-                <Route exact path="/messages" component={Messages} />
-                <Route exact path="/dashboard">
-                  <Dashboard />
-                </Route>
-                <Route path="/booking" component={Booking} />
-                <Route path="*">
-                  <Redirect to="/login" />
-                </Route>
-              </Switch>
+              <NotificationProvider>
```

Comment:
Optional: I would advise against wrapping the entire app with the `NotificationProvider` - each time we get a new notification the entire app will re-render which will significantly affect app performance :)

Reply:
@rajivtitus  Do you suggest removing it? or wrap only the component that needs it?

Reply:
@hayderjebur Since you have already created it, you can wrap only the component that needs it.